common/djangoapps/student/models.py

```diff
@@ -1863,7 +1865,7 @@ def _cert_name(self, course_name, cert_mode):
             cert_mode,
             _(u"{platform_name} Certificate for {course_name}")
         ).format(
-            platform_name=settings.PLATFORM_NAME,
+            platform_name=microsite.get_value('platform_name', settings.PLATFORM_NAME),
```

Comment:
Is there a platform name accessor available which abstracts these two parameters?  Or maybe it was just added to static_content?  Would be useful to have something like that available within the system, as well.